### PR TITLE
Various documentation updates

### PR DESCRIPTION
**Pull request by alsrgv**
1. Update LF DL logo to LF AI logo
2. Revived a page about `mpirun` direct usage (cc @kit1980)
3. Add #1040 to troubleshooting
4. Added information about resolving `bash: orted not found` error (see #1032 and few other tickets)

cc @IgorWilbert @sblotner

### CODE REVIEW

File: docs/mpirun.rst [hunk 1]
```diff
@@ -0,0 +1,153 @@
+Running Horovod with Open MPI
+=============================
+
+``horovodrun`` introduces a convenient wrapper for running Horovod scripts, that is currently based on Open MPI.
```
Comment (sblotner) [inline docs/mpirun.rst]: a convenient, Open MPI-based wrapper for running Horovod scripts.

File: docs/mpirun.rst [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+In some cases it is desirable to have a fine-grained control over options passed to Open MPI.  This page describes
```
Comment (sblotner) [inline docs/mpirun.rst]: have a --> have

File: docs/mpirun.rst [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        -mca pml ob1 -mca btl ^openib \
+        python train.py
+
+Starting with the Open MPI 3, it's important to add the ``-bind-to none`` and ``-map-by slot`` arguments.
+``-bind-to none`` specifies Open MPI to not bind a training process to a single CPU core (which would hurt performance).
+``-map-by slot`` allows you to have a mixture of different NUMA configurations because the default behavior is to bind
+to the socket.
+
+``-mca pml ob1`` and ``-mca btl ^openib`` flags force the use of TCP for MPI communication.  This avoids many
+multiprocessing issues that Open MPI has with RDMA which typically result in segmentation faults.  Using TCP for MPI
```
Comment (sblotner) [inline docs/mpirun.rst]: result --> results

File: docs/mpirun.rst [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+This is frequently useful in the case of `running Horovod in Docker environment <docker.md>`_.
+
+Open MPI with RDMA
+------------------
+
+As noted above, using TCP for MPI communication does not have any significant effects on performance in the majority of
+cases. Models that make heavy use of ``hvd.broadcast()`` and ``hvd.allgather()`` operations are exceptions to that rule.
+
+Default Open MPI ``openib`` BTL that provides RDMA functionality does not work well with MPI multithreading.  In order
+to use RDMA with ``openib``, multithreading must be disabled via ``-x HOROVOD_MPI_THREADS_DISABLE=1`` option.  See the
```
Comment (sblotner) [inline docs/mpirun.rst]: via --> via the

File: docs/mpirun.rst [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+If you see non-routed interfaces (like ``docker0``) in the output of ``ifconfig``, you should tell Open MPI and NCCL to not
+use them via the ``-mca btl_tcp_if_exclude <interface>[,<interface>]`` and ``NCCL_SOCKET_IFNAME=^<interface>[,<interface>]``
+parameters.
+
+.. code-block:: bash
+
+    ifconfig
+
+Produces output like this::
```
Comment (sblotner) [inline docs/mpirun.rst]: you might need to add a space after "this"

Comment (alsrgv) [reply docs/mpirun.rst]: AFAIK space is not necessary - http://docutils.sourceforge.net/docs/user/rst/quickstart.html#preformatting-code-samples

File: README.rst [hunk 6]
```diff
@@ -182,26 +182,23 @@ page for more instructions, including RoCE/InfiniBand tweaks and tips for dealin
...
 
-
 2. To run on 4 machines with 4 GPUs each:
 
-.. code-block:: python
+.. code-block:: bash
 
     horovodrun -np 16 -H server1:4,server2:4,server3:4,server4:4 python train.py
 
+3. To run using Open MPI without ``horovodrun`` wrapper, see the `Running Horovod with Open MPI <docs/mpirun.rst>`_ page.
```
Comment (sblotner) [inline README.rst]: without --> without the

File: README.rst [hunk 7]
```diff
@@ -182,26 +182,23 @@ page for more instructions, including RoCE/InfiniBand tweaks and tips for dealin
...
-
-
-4. To run in Kubernetes, see `Kubeflow <https://github.com/kubeflow/kubeflow/tree/master/kubeflow/mpi-job>`_, `MPI Operator <https://github.com/kubeflow/mpi-operator/>`_, `Helm Chart <https://github.com/kubernetes/charts/tree/master/stable/horovod/>`_, and `FfDL <https://github.com/IBM/FfDL/tree/master/etc/examples/horovod/>`_.
+4. To run in Docker, see the `Horovod in Docker <docs/docker.md>`_ page.
 
+5. To run in Kubernetes, see `Kubeflow <https://github.com/kubeflow/kubeflow/tree/master/kubeflow/mpi-job>`_, `MPI Operator <https://github.com/kubeflow/mpi-operator/>`_, `Helm Chart <https://github.com/kubernetes/charts/tree/master/stable/horovod/>`_, and `FfDL <https://github.com/IBM/FfDL/tree/master/etc/examples/horovod/>`_.
 
-
-5. To run in Spark, see the `Spark <docs/spark.md>`_ page.
+6. To run in Spark, see the `Spark <docs/spark.md>`_ page.
```
Comment (sblotner) [inline README.rst]: This page doesn't exist yet in RTD, right?

Comment (alsrgv) [reply README.rst]: Not yet, we should convert `.md` to `.rst`

**Comment by alsrgv** [comment]
@sblotner, thanks!  I've added one more answer to troubleshooting guide, could you take a look?

File: docs/troubleshooting.md [hunk 8]
```diff
@@ -331,3 +331,46 @@ $ apt purge hwloc-nox libhwloc-dev libhwloc-plugins libhwloc5
...
+  one of the contrib/platform definitions for your system type.
+
+* an inability to create a connection back to mpirun due to a
+  lack of common network interfaces and/or no route found between
+  them. Please check network connectivity (including firewalls
+  and network routing requirements).
+--------------------------------------------------------------------------
+```
+
+We recommended reinstalling Open MPI with `--enable-orterun-prefix-by-default` flag, like so:
```
Comment (sblotner) [inline docs/troubleshooting.md]: with --> with the

**Pull request merged**